SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -1,12 +1,13 @@
-# Port your Unreal project to the GDK
+# Get started: Port your own Unreal project to the GDK
```

Comment:
General comment - this still uses "game project" (instead of "project") in the "Terms used in this guide" section. But I can't edit it in this PR because it's not something you've changed.

Reply:
kk

Reply:
dOne